src/sagemaker/fw_utils.py

```diff
@@ -126,6 +126,28 @@ def _accelerator_type_valid_for_framework(framework, accelerator_type=None, opti
     return True
 
 
+def validate_git_config(git_config):
```

Comment:
Maybe fw_utils is not the best file for this validation. You could leave it in the estimator or create a new file `git.py`.

Reply:
I leave it in the estimator.